integration_tests/client/metrics/test_classification.py

```diff
@@ -810,8 +810,660 @@ def test_evaluate_classification_with_label_maps(
     }
 
     cat_expected_metrics = [
+        {"type": "Accuracy", "parameters": {"label_key": "k3"}, "value": 0.0},
+        {"type": "ROCAUC", "parameters": {"label_key": "k3"}, "value": 1.0},
+        {
+            "type": "PrecisionRecallCurve",
+            "parameters": {"label_key": "k3", "pr_curve_iou_threshold": None},
+            "value": {
+                "v1": {
+                    "0.1": {
```

Comment:
the confidence threshold keys are stored as strings instead of floats in psql. this isn't ideal since it causes these keys to be out-of-order when viewed via the Python client, but I didn't see any easy workarounds to this issue given we're using the JSONB field type. I'd recommend that we leave as-is and let the front-end team convert these keys to floats on their end, but open to your feedback